tests/unit/components/status/StatusBar.spec.js

```diff
@@ -84,4 +98,75 @@ describe("StatusWidget.vue", () => {
       "Status:3dbb8814-09f1-44db-b7d5-7a9f702beac4"
     );
   });
+  test("When Vuex is mutated to an ERROR UUID, Then the status text should update as 'Error Occurred' and the the dialog of ErrorCatchWidget is visible ", async () => {
+    const shutdown_url = "http://localhost:4567/shutdown";
+    mocked_axios.onGet(shutdown_url).reply(200, {});
+    const propsData = {};
+    wrapper = mount(StatusWidget, {
+      propsData,
+      store,
+      localVue,
+      attachToDocument: true,
+    });
+
+    expect(wrapper.contains("#error-catch")).toBe(true);
+    const modal = wrapper.find("#error-catch");
+    expect(modal.isVisible()).toBe(false);
+    store.commit("flask/set_status_uuid", STATUS.MESSAGE.ERROR);
+    await wrapper.vm.$nextTick(); // wait for update
+    expect(wrapper.find(text_selector).text()).toEqual(
+      "Status: Error Occurred"
+    );
+    Vue.nextTick(() => {
+      expect(modal.isVisible()).toBe(true);
+      done();
+    });
+  });
+  test("When Vuex is mutated to an SHUTDOWN UUID, Then the status text should update as 'Shutting Down' ", async () => {
+    const propsData = {};
+    wrapper = mount(StatusWidget, {
+      propsData,
+      store,
+      localVue,
+    });
+
+    store.commit("flask/set_status_uuid", STATUS.MESSAGE.SHUTDOWN);
+    await wrapper.vm.$nextTick(); // wait for update
+    expect(wrapper.find(text_selector).text()).toEqual("Status: Shutting Down");
+    await wrapper.vm.$nextTick(); // wait for update
+  });
+  test("When an event 'ok-clicked'  is emitted from 'ErrorCatchWidget, Then verify that the dialog of ErrorCatchWidget is hidden", async () => {
```

Comment:
You also have an `expect` statement about the Text. Please include something describing that in the `Then` section of the test title

Reply:
Added 'Status is changed to 'Shutting Down'.